src/.eslintrc.js

```diff
@@ -4,5 +4,14 @@ module.exports = {
   ignorePatterns: ['three-line2'],
   env: {
     browser: true
+  },
+  rules: {
+    'no-unused-vars': [
+      'error',
+      {
+        argsIgnorePattern: '^_',
+        ignoreRestSiblings: true
```

Comment:
That's how I got rid of the es-lint disable comments when defining a class properties without using them in the constructor. Typescript implicitly uses Rest to assign the values, if I understood correctly.